src/api/sentiment.js

```diff
@@ -1,75 +1,80 @@
-const Twit = require('twit')
-const config = require('../config')
-
-const bot = new Twit(config.twitterKeys)
-
 const quote = require('positivity-api')
 
-const db = require('../helpers/db')
-
 // sentiment deps
 const unified = require('unified')
 const sentiment = require('retext-sentiment')
 const english = require('retext-english')
 
+const bot = require('../twitBot')
+
+const db = require('../helpers/db')
+
 // event is passed in from calling bot.js function
 const sentimentBot = event => {
-  // console.log('====================')
-  // console.log(event)
-  // console.log('====================')
   const processor = unified()
     .use(english)
     .use(sentiment)
   const tree = processor.parse(event.text)
 
   processor.run(tree, event.text)
 
+  // eslint-disable-next-line no-console
   console.log('====================}')
+  // eslint-disable-next-line no-console
   console.log(`POLARITY=${tree.data.polarity}`)
+  // eslint-disable-next-line no-console
   console.log(`VALENCE=${tree.data.valence}`)
+  // eslint-disable-next-line no-console
   console.log(`TWEET TEXT=${event.text}`)
+  // eslint-disable-next-line no-console
   console.log('====================')
 
-  const polarity = tree.data.polarity
-  const valence = tree.data.valence
+  const { polarity, valence } = tree.data
 
   // Don't do anything if it's the bot tweet
-  if (event.user.screen_name == '_100DaysOfCode') return
+  if (event.user.screen_name === '_100DaysOfCode') return
 
   // if polarity is negative and polarity is <= -2
-  if (valence == 'negative' && polarity <= -2) {
+  if (valence === 'negative' && polarity <= -2) {
     // get a random quote
     const phrase = quote.random()
-    const screen_name = event.user.screen_name
+    const screenName = event.user.screen_name
     const tweetId = event.id_str
 
     // Check key isn't in db already, key being the screen_name
-    db.get(screen_name, (err, value) => {
+    db.get(screenName, (_, value) => {

```

Comment:
For my own education here what is the underscore `_` for?

Reply:
Here this is because the first argument is not use. If we put a argument name eslint will not be happy. By putting underscore here eslint dont complain anymore